chatcommunicate.py

```diff
@@ -59,7 +58,10 @@ def special_room_watcher(ev, wrap2):
         post_site_id = fetch_post_id_and_site_from_msg_content(content_source)
         post_url = fetch_post_url_from_msg_content(content_source)
         if post_site_id is not None and post_url is not None:
-            t_check_websocket = Thread(target=DeletionWatcher.check_if_report_was_deleted, args=(post_site_id, post_url, ev.message))
```

Comment:
I don't share the same opinion about the `Thread(target=` style indenting on one line, but if that's the way you want it